src/v2/providers/alerts/appDistribution.ts

```diff
@@ -1,10 +1,10 @@
-import { defineEndpoint, FirebaseAlertData } from '.';
-import { ManifestEndpoint } from '../../../common/manifest';
+import { getEndpointAnnotation, FirebaseAlertData } from './alerts';
 import { CloudEvent, CloudFunction } from '../../core';
 import * as options from '../../options';
 
-/** Data */
-/** Payload for a new tester device */
+/**
+ * The internal payload object that is wrapped inside a FirebaseAlertData object
```

Comment:
For consistency, I think we should end each comment with a period `.`

Reply:
Added periods to public comments